docs/store-operations/catalog/graphql-admin/product-images.mdx

```diff
@@ -0,0 +1,510 @@
+# Product Images (Beta)
+
+[_International Enhancements for Multi-Storefront_](/docs/store-operations/catalog/msf-international-enhancements)
+
+<Callout type="info">
+  - This feature is currently available for Enterprise stores and Partner Sandboxes. If the feature is not working as expected, please contact technical support, as the feature likely needs to be enabled for the individual store.
+  To become an enterprise customer, contact your BigCommerce Customer Service Manager or our [support team](https://support.bigcommerce.com/s/contact?language=en_US).
+  - We may introduce changes to this beta feature without notice.
+</Callout>
+
+Using the Catalog features of the Admin API, you can set and query information for product images.
+
+First, add the image to the product for the global store. You can do so using the [Create a product image](/docs/rest-catalog/products/images#create-a-product-image) endpoint of the REST Management API. 
+
+You can then [add the image to the product](#add-image-to-product) for a channel locale and perform the following: 
```

Comment:
`addImagesToProduct ` and `removeImagesFromProduct` also works with global level. These commands without `context` manage `addedToProduct` property on global level.

Reply:
@alex-gaiduchok If I'm using `addImagesToProduct` to add an image to the product for the _global level_, where can I get the image ID to use in the input? I was thinking they could get the image ID from the [Get all Product Images](https://developer.bigcommerce.com/docs/rest-catalog/products/images) endpoint, but the endpoint returns the images that have _already_ been added the product

I asked the same question [here](https://bigcommerce.slack.com/archives/CGC6V5C0H/p1728334116910269?thread_ts=1727287753.845109&cid=CGC6V5C0H) (but I think the team might have misunderstood my question earlier?).

Reply:
ok, now I understand, when we are talking about `addedToProduct` we mean image property which is just a boolean flag and it affects only image displaying on storefront. Each uploaded image have this flag set as `true` by default. If you want to hide some image from storefront but keep it for some sales or holidays you can set `addedToProduct` to `false` by `removeImagesFromProduct`, you still can find this image into `images` node, its not deleted from product just removed from displaying. Hence firstly image should be uploaded and only then we can manage `addedToProduct` flag because `images` with particular ID is exists

Reply:
Ah, got it, thanks @alex-gaiduchok. I've added the information I was missing 👍